scripts/entrypoint.pl

```diff
@@ -174,8 +315,6 @@ sub write_localconfig {
         }
     }
 
-    unlink($filename);
```

Comment:
We no longer unlink, relying on the container to start without that file existing.
In the tests we have to ensure it is deleted before calling the entrypoint (and it is an obvious error if we forget). I'll document that behavior in the readme in a later PR.